run.sh

```diff
@@ -4,7 +4,8 @@ export CLUSTER_NAME=${CLUSTER_NAME:=elasticsearch}
 export NODE_NAME=${NODE_NAME:=${HOSTNAME}}
 export NODE_LOCAL=${NODE_LOCAL:=false}
 export PATH_DATA=${PATH_DATA:=/data}
-export ES_HEAP_SIZE=${ES_HEAP_SIZE:=450m}
+export ELASTIC_SEARCH_HEAP_SIZE=${ELASTIC_SEARCH_HEAP_SIZE:=450m}
```

Comment:
`ES_HEAP_SIZE` is no longer supported and ES complaints about it as head size is now set via java opts. Renamed to keep it separately configurable.